api/src/neo4j/index.js

```diff
@@ -34,4 +32,5 @@ export {
   getInteractionPartners,
   getMapsListing,
   mapSearch,
+  search,
```

Comment:
For https://github.com/MetabolicAtlas/MetabolicAtlas/pull/508#discussion_r481805539:

I actually renamed the original `search` function into `_search` and added a new `search` function, which is: `const search = async (params) => params.model ? modelSearch(params) : globalSearch(params);
`. It helped to clean things up in the `api/src/endpoints/neo4j.js` file. So no change in functionality here, and the results are still different depending on if a model is provided.